registry/src/store/reducers/searchReducer.js

```diff
@@ -32,6 +33,13 @@ const searchReducer = (state = initialState, action) => {
         currentPage: action.payload.currentPage,
       };
 
+      case SEARCH_PACKAGE:
```

Comment:
Is this still required if we are having ``SEARCH_LOADING`` already https://github.com/fortran-lang/registry/pull/28/files#diff-b99141865f41a611be4f7bb8bf449a9f5b8483232ae0b2aed07c3381d56eb4e1R7 ?